### ISSUE TITLE
Nest 9.X support?

### ISSUE THREAD

**Issue by ricbermo** [rejvban/nestjs-twilio#37]
# Bug Report

## Describe the Bug

This package cannot be installed on NestJS 9 projects

## How to Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:
1. try to install this package on a NestJS 9 project.

## Expected Behavior

Tell me what should happen.

## Error

Error resulted by the potential bug.

## Your Environment

- OS: [e.g. macOS, Windows]
- Version of nestjs-twilio: [e.g. v1.0.0]

## Error reproducing steps

Please explain how did your error ocurr, you can also leave gists, repos or any kind of codebase.

## Additional Information

Any other information about the problem here.

**Comment by ricbermo** [comment]
Sorry I did not include all details, IMHO the title is pretty self explanatory 😅

Happy to help if needed!!

**Comment by lkaric** [comment]
@ricbermo, I'll be releasing a major version compatible with Nestjs 9.X in a couple of days. Thanks for showing interest in contributing!

**Comment by thisisomar** [comment]
Any updates on this? Happy to help if needed :)

**Comment by lkaric** [comment]
@thisisomar Just an Update, I'm cutting down the code and making it more maintainable by utilizing the [configurable module builder](https://docs.nestjs.com/fundamentals/dynamic-modules#configurable-module-builder). For the time being, I'm deprecating `@InjectTwilio` decorator and instead, it will be used as a service until I figure out how to provide the decorator as well. I'm publishing the new major version tonight.

Change in usage

```ts
@InjectTwilio()
private readonly twilioClient: TwilioClient
```
will change to
```ts
private readonly twilioService: TwilioService
```

If you're familiar with any approach that could provide us with a service approach and decorator, I'd be glad to hear it and/or look at any pr requests coming my way.

**Comment by lkaric** [comment]
@thisisomar @ricbermo Released [v3.0.0](https://github.com/rejvban/nestjs-twilio/releases/tag/v3.0.0) a couple of minutes ago, be sure to check out the docs because of the breaking changes. #38

**Comment by lkaric** [comment]
@ricbermo Closing the issue, considering it's resolved with v3
